config/default.json

```diff
@@ -31,5 +31,8 @@
         "maxazs": 2,
         "cidrMask": 20
     },
-    "defaultremovalpolicy": "snapshot"
+    "defaultremovalpolicy": "snapshot",
+    "bucketname": "admin.nelson.management",
```

Comment:
Please name this property something meaningful. We will have many buckets. We need to know what we are using this bucket for. Same for the lines below as well. Maybe move them into a object like 
```
...,
"nelsonsaasbucket": {
"bucketname": "admin.nelson.management",
"useexistingbucket": true,
"publicreadaccess": true
},
...
```

Reply:
Resolved. Can you help to re-check it? @sandeepbaynesnelson